### DISCUSSION TITLE
Releasing 2.0.0.xxxx versions

### DISCUSSION THREAD

**Discussion by GayanSandaruwan** [cucumber/cucumber-eclipse#453] [General]
Some time back We saw the 2.0.0 had hit the Eclipse market place, and installed it and got used to. But Don't see it there anymore. Also the released versions in releases only contain 1.0.0.xxx versions only.
I created this draft release. What do you think ?

https://github.com/cucumber/cucumber-eclipse/releases/tag/untagged-f578e8788204b95c64a8

**Comment by laeubi** [comment]
You need to add the Update-site to eclipse, "releasing" it on Github has no effect for this, the current snapshot page is:

https://cucumber.github.io/cucumber-eclipse/update-site/main/

so just add this to your eclipse (we need to update some of the docs I think they seem to reference old pages) and after every build this gets updated.

**Reply by GayanSandaruwan** [reply]
@laeubi  Thank you for the response. The update site didn't seem to work for quite some time. At least for me. That's when I started looking for the releases in the GitHub releases. 
Tested now, can install the new version from update site. Thank you for the info.

Anyway, what do you think of adding a release in github so that, someone looking to install by archive, they can find it here.

**Reply by laeubi** [reply]
> Tested now, can install the new version from update site. Thank you for the info.

I have updated the link on the main readme so probably before you used the old (non working) link ;-)

> Anyway, what do you think of adding a release in github so that, someone looking to install by archive, they can find it here.

I think the Update-site is always preferable as it allows to stay up-to-date with bugfixes, the github releases are more useful for a stable release, not publishing snapshots.

Sadly it seems not much people where using the 2.x yet and just complain about "old" bugs but do not contribute much here, so probably the user-base is quite low.

That said, if you think you can test and maybe update the docs a bit we can just make a new 2.0.0 stable release after that.
